tests/testthat/_snaps/write.md

```diff
@@ -0,0 +1,11 @@
+# Combining skimr, labelled, and xlsx returns expected snapshot
```

Comment:
I would expect much more data in this snap. I don't think this will be enough to capture important problems.

Reply:
Ah yes sorry, this was an oversight. There are plenty of sub-objects in the `wb` output (including styles, relationships, etc.) , but I think what we're really interested in is `wb_obj$tables`:

```r
> wb_obj$tables %>% tibble()
# A tibble: 11 × 5
   tab_name tab_sheet tab_ref tab_xml                                                                        tab_act
   <chr>        <int> <chr>   <chr>                                                                            <dbl>
 1 Table1           1 A2:H12  "<table xmlns=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\" x…       1
 2 Table2           2 A2:D6   "<table xmlns=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\" x…       1
 3 Table3           3 A2:D6   "<table xmlns=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\" x…       1
 4 Table4           4 A2:E6   "<table xmlns=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\" x…       1
 5 Table5           5 A2:AE6  "<table xmlns=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\" x…       1
 6 Table6           6 A2:R6   "<table xmlns=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\" x…       1
 7 Table7           7 A2:C6   "<table xmlns=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\" x…       1
 8 Table8           8 A2:E6   "<table xmlns=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\" x…       1
 9 Table9           9 A2:I6   "<table xmlns=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\" x…       1
10 Table10         10 A2:J5   "<table xmlns=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\" x…       1
11 Table11         11 A2:AZ68 "<table xmlns=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\" x…       1
```

Do you think we need to capture the entire R6 output contents? Looking through the docs to see if its possible to snapshot environments but it we may do better snapshotting elements of interest instead.

Reply:
That's a good question. I think it depends on how sensitive we want to be to subtle changes. We can discuss at standup.

Reply:
I don't see the data or the labels in this snapshot so it doesn't seem to capture all of the data we want to keep track of.

Reply:
I found a nice function [`openxlsx2::wb_to_df()`](https://janmarvin.github.io/openxlsx2/reference/wb_to_df.html) that I think we can `map` with and add the dataframe outputs to this list snapshot. Implementing now for upcoming commit, but I think it looks nice and has all the data now.

The actual data is buried pretty deep in the `wb` object and from what I could tell wasn't very readable:

```r
> wb_obj$worksheets[[1]]$sheet_data$cc_out %>% tibble()
# A tibble: 96 × 15
   row_r c_r   r     v     c_t       c_s   c_cm  c_ph  c_vm  f     f_t   f_ref f_ca  f_si  is                                           
   <chr> <chr> <chr> <chr> <chr>     <chr> <chr> <chr> <chr> <chr> <chr> <chr> <chr> <chr> <chr>                                        
 1 1     A     A1    ""    inlineStr "5"   ""    ""    ""    ""    ""    ""    ""    ""    <is><t>REDCap Instrument Name</t></is>       
 2 1     B     B1    ""    inlineStr "5"   ""    ""    ""    ""    ""    ""    ""    ""    <is><t>REDCap Instrument Description</t></is>
 3 1     C     C1    ""    inlineStr "5"   ""    ""    ""    ""    ""    ""    ""    ""    <is><t>Repeating or Nonrepeating?</t></is>   
 4 1     D     D1    ""    inlineStr "5"   ""    ""    ""    ""    ""    ""    ""    ""    <is><t># of Rows in Data</t></is>            
 5 1     E     E1    ""    inlineStr "5"   ""    ""    ""    ""    ""    ""    ""    ""    <is><t># of Columns in Data</t></is>         
 6 1     F     F1    ""    inlineStr "5"   ""    ""    ""    ""    ""    ""    ""    ""    <is><t>Data size in Memory</t></is>          
 7 1     G     G1    ""    inlineStr "5"   ""    ""    ""    ""    ""    ""    ""    ""    <is><t>% of Data Missing</t></is>            
 8 1     H     H1    ""    inlineStr "5"   ""    ""    ""    ""    ""    ""    ""    ""    <is><t>Sheet #</t></is>                      
 9 2     A     A2    ""    inlineStr ""    ""    ""    ""    ""    ""    ""    ""    ""    <is><t>redcap_form_name</t></is>             
10 2     B     B2    ""    inlineStr ""    ""    ""    ""    ""    ""    ""    ""    ""    <is><t>redcap_form_label</t></is>            
# ℹ 86 more rows
# ℹ Use `print(n = ...)` to see more rows
```